examples/with-supabase/config/backendConfig.js

```diff
@@ -0,0 +1,61 @@
+import ThirdPartyEmailPasswordNode from "supertokens-node/recipe/thirdpartyemailpassword";
+import SessionNode from "supertokens-node/recipe/session";
+import { appInfo } from "./appInfo";
+
+export let backendConfig = () => {
+    return {
+        framework: "express",
+        supertokens: {
+            connectionURI: "https://try.supertokens.io",
+        },
+        appInfo,
+        recipeList: [
+            ThirdPartyEmailPasswordNode.init({
+                providers: [
+                    // We have provided you with development keys which you can use for testing.
+                    // IMPORTANT: Please replace them with your own OAuth keys for production use.
+                    ThirdPartyEmailPasswordNode.Google({
+                        clientId: process.env.GOOGLE_CLIENT_ID,
+                        clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+                    }),
+                    ThirdPartyEmailPasswordNode.Github({
+                        clientId: process.env.GITHUB_CLIENT_ID,
+                        clientSecret: process.env.GITHUB_CLIENT_SECRET,
+                    }),
+                    ThirdPartyEmailPasswordNode.Apple({
+                        clientId: process.env.APPLE_CLIENT_ID,
+                        clientSecret: {
+                            keyId: process.env.APPLE_KEY_ID,
+                            privateKey: process.env.APPLE_PRIVATE_KEY.replace(/\\n/g, "\n"),
+                            teamId: process.env.APPLE_TEAM_ID,
+                        },
+                    }),
+                ],
+            }),
+            SessionNode.init({
+                override: {
+                    functions: (originalImplementation) => {
+                        return {
+                            ...originalImplementation,
+                            // We override the createNewSession function so we can store supabase's JWT secret signing key
+                            // so it can be used on the frontend
+                            createNewSession: async function (input) {
+                                let response = await originalImplementation.createNewSession(input);
+                                let currentAccessTokenPayload = await response.getAccessTokenPayload();
+
+                                // update the accessTokenPayload to include the supabase secret
+                                await response.updateAccessTokenPayload({
+                                    ...currentAccessTokenPayload,
+                                    supabase_secret: process.env.SUPABASE_SIGNING_SECRET,
```

Comment:
- you are supposed to create the signed JWT for the user for whom you are creating the token.
- Add that token to the argument when you call `originalImplementation.createNewSession`